### PR TITLE
Slaierno multilanguage

### PR DESCRIPTION
**Pull request by slaierno**
Multilanguage support added! Yay!

It was a little bit trick, but it should work. It's not..._nice_, I would say, so if anyone wants to modify the appereances (that footer is a little bit plain, I know) it would be appreciated!

Please, check if anything I modified from your pages (@kmfrick only, actually, since he was the only one with a multilanguage page) is fine.

Also, let me know if anything is clear and if you manage to create multilanguage pages.

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -49,8 +49,7 @@ Content of my personal page in _markdown_ **syntax**
 * `reddit`: your reddit username
 * `langs`, `frameworks` and `misc`: your skills, refer to the next section.
 
-`layout`, `short_name` and `name` are mandatory. Everything else is optional.
-
+`layout`, `short_name` and `name` are mandatory. Everything else is optional. Note that in case of **multilanguage author pages**, only this three mandatory variables and the language-specific ones (`lang` and `lang-ref`) are necessary. Everything else will be retrieved from the default-language (i.e. Italian) author page.
```
Comment (kmfrick) [inline README.MD:52]: s/this three/these three

File: README.MD [hunk 2]
```diff
@@ -122,6 +121,22 @@ You can then add equations by enclosing them in double dollar signs, like so: $$
 
 An example post where you can refer is `_posts/example.markdown`.
 
+## Multilanguage support
+
+The blog support multiple languages for pretty much everything. The defuault language is Italian, for every other language you may:
```
Comment (kmfrick) [inline README.MD:126]: s/defuault/default

File: README.MD [hunk 3]
```diff
@@ -122,6 +121,22 @@ You can then add equations by enclosing them in double dollar signs, like so: $$
...
+* add a subfolder (project root included) named with the language short name (refer to `_data/languages.yml)
+* add to the frontmatter two variables, `lang`, indicating the languages, and `lang-ref`, representing a label for that page.
+
+Posts will be listed on a language base. E.g. in you english author page you will not see your non-english posts.
```
Comment (kmfrick) [inline README.MD:130]: s/base/basis

File: README.MD [hunk 4]
```diff
@@ -122,6 +121,22 @@ You can then add equations by enclosing them in double dollar signs, like so: $$
...
+
+A user can change the language for a page (if available) via the links on the footer of suche page.
```
Comment (kmfrick) [inline README.MD:132]: s/suche/such

File: README.MD [hunk 5]
```diff
@@ -122,6 +121,22 @@ You can then add equations by enclosing them in double dollar signs, like so: $$
...
+
+NOTE: multilanguage support is tricky. If you note anything wrong, please issue a bug!
```
Comment (kmfrick) [inline README.MD:134]: s/note/notice

File: _includes/skill_table.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+{%- assign langs_framework_misc = "langs,frameworks,misc" | split: "," -%}
+{%- assign self_want = "self,want" | split: "," -%}
+
+{% for stype in langs_framework_misc -%}
+
+<h1> {{ site.data.skillset["titles"][stype] }} </h1>
+
+    {%- for skill_ in site.data.skillset[stype] -%}
+        {%- assign skill_name = skill_[0] -%} {%- comment -%} skill name we are searching for {%- endcomment -%}
+        {%- assign skill = skill_[1] -%} {%- comment -%} possible skill-related skills {%- endcomment -%}
```
Comment (kmfrick) [inline _includes/skill_table.md:13]: "skill related skills"?

File: _data/translations.yml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+
+differentLanguage:
+  en: "Read this page in a different language"
+  it: "Leggi questa pagina in un altra lingua"
+  jp: "このページを別の言語で読む"
+viewIn:
+  en: "View in"
+  it: "Guarda in"
+  it: "で見る"
```
Comment (kmfrick) [inline _data/translations.yml:9]: s/it/jp

File: _includes/skill_table.md (hunk 6 above)
Comment (slaierno) [reply _includes/skill_table.md:13]: Ahhhh, how should I phrase that? I made an awful job naming variables here...

Comment (kmfrick) [reply _includes/skill_table.md:13]: `skill_[1]` is the "vers" variable, right? So maybeeee "possible multiple versions of this skill"? But I'm not sure

**Pull request merged**